### ISSUE TITLE
Return information regarding image-mask mapping 

### ISSUE THREAD

**Issue by Nelson-Gon** [MouseLand/cellpose#582]
Hi MouseLand team,

Thank you for this awesome work. I am exploring the package and was wondering if there could be a way to return a verbose message regarding which image/mask was used in the `model.eval` step.

Given this example from the documentation for example 

```
import numpy as np
import matplotlib.pyplot as plt
from cellpose import models
from cellpose.io import imread

# model_type='cyto' or 'nuclei' or 'cyto2'
model = models.Cellpose(model_type='cyto')

# list of files
# PUT PATH TO YOUR FILES HERE!
files = ['/media/carsen/DATA1/TIFFS/onechan.tif']

imgs = [imread(f) for f in files]
nimg = len(imgs)

# define CHANNELS to run segementation on
# grayscale=0, R=1, G=2, B=3
# channels = [cytoplasm, nucleus]
# if NUCLEUS channel does not exist, set the second channel to 0
channels = [[0,0]]
# IF ALL YOUR IMAGES ARE THE SAME TYPE, you can give a list with 2 elements
# channels = [0,0] # IF YOU HAVE GRAYSCALE
# channels = [2,3] # IF YOU HAVE G=cytoplasm and B=nucleus
# channels = [2,1] # IF YOU HAVE G=cytoplasm and R=nucleus

# if diameter is set to None, the size of the cells is estimated on a per image basis
# you can set the average cell `diameter` in pixels yourself (recommended)
# diameter can be a list or a single number for all images

masks, flows, styles, diams = model.eval(imgs, diameter=None, channels=channels)

```

I think it would be great to have information that maps `masks` back to `imgs`. I thought I could get this information in the styles array.

Why would this information be important?

I think it would help one to quickly compare these masks to some known mask, visually and ensure that whatever images they are comparing are in fact a matching pair. 

My vision of the implementation might be something like 

```
masks, ... = model.eval(verbose=True)
```

Then verbose message would tell me for example that

```
Mask [index] for [image_file_path] created. 
```
Apologies if this already exists in the docs.

Thank you,
NelsonGon

**Comment by carsen-stringer** [comment]
this doesn't exist, but I think we don't want to add this kind of info because it's a bit confusing for users running on the command line -- the masks there are saved as `filename_masks.tif` so the indexing would be confusing
